### PR TITLE
fix(manager/helm-values): Revert "feat(manager/helm-values): Add support for bumpVersion"

### PR DESCRIPTION
**Pull request by rarkins**
Reverts renovatebot/renovate#26441

### CODE REVIEW

File: docs/usage/configuration-options.md [hunk 1]
```diff
@@ -406,14 +406,13 @@ Instead, set the old `branchPrefix` value as `branchPrefixOld` to allow Renovate
 ## branchTopic
 
 This field is combined with `branchPrefix` and `additionalBranchPrefix` to form the full `branchName`. `branchName` uniqueness is important for dependency update grouping or non-grouping so be cautious about ever editing this field manually.
-This is an advanced field, and it's recommend you seek a config review before applying it.
+This is an advance field and it's recommend you seek a config review before applying it.
```
Comment (viceice) [inline docs/usage/configuration-options.md:409]:
```suggestion
This is an advanced field, and it's recommend you seek a config review before applying it.
```
let's leave this improvement

**Review by viceice — COMMENTED** [review]
otherwise LGTM

**Review by viceice — APPROVED** [review]

**Comment by maybeanerd** [comment]
Why was this reverted?

**Comment by viceice** [comment]
> Why was this reverted?

because it's broken.

**Pull request merged**